imports/components/player_list.jsx

```diff
@@ -0,0 +1,44 @@
+export class PlayerList extends React.Component {
+  render() {
+    var players = this.props.players.map(function(player) {
+      return (<PlayerListEntry key={player._id} {...player} />);
+    });
+    var logIn = null;
+    if (!this.props.loggedIn) {
+      logIn = <li>
+        <strong>Log In</strong> to join this list
+      </li>
```

Comment:
Oh, and this is the package.json diff I ended up with when I was doing some quick prototyping:

```
+    "eslint": "^3.19.0",
+    "eslint-import-resolver-meteor": "^0.4.0",
+    "eslint-plugin-jsx-a11y": "^5.1.0",
+    "eslint-plugin-meteor": "^4.0.1",
```